core/token.go

```diff
@@ -22,16 +22,21 @@ import (
 	"io/ioutil"
 )
 
-type Secret struct {
-	Token string `json:"root_token"`
+type Auth struct {
+	Secret Inner `json:"auth"`
 }
 
-func getSecret(filename string) (Secret, error) {
-	s := Secret{}
+type Inner struct {
```

Comment:
Does this Inner have to be exposed outside of this `core` package? We can use small `inner` if possible.

Reply:
I will close this PR as this has been address by #46.